{"uuid": "b924420e-072f-4c11-ac11-d6e79861b647", "children": ["2a2fabae-19c7-4e6e-a37d-cca10e585ef8", "df6d42dc-0167-48e0-b028-7af946c1ec87"], "befores": [{"name": "init", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.NoSuchWindowException: Message: no such window: target window already closed\nfrom unknown error: web view not found\n  (Session info: chrome=96.0.4664.110)\n\n", "trace": "  File \"C:\\Users\\bonjour\\AppData\\Local\\Programs\\Python\\Python39\\lib\\site-packages\\pluggy\\_callers.py\", line 39, in _multicall\n    res = hook_impl.function(*args)\n  File \"C:\\Users\\bonjour\\AppData\\Local\\Programs\\Python\\Python39\\lib\\site-packages\\_pytest\\fixtures.py\", line 1126, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"C:\\Users\\bonjour\\AppData\\Local\\Programs\\Python\\Python39\\lib\\site-packages\\_pytest\\fixtures.py\", line 925, in call_fixture_func\n    fixture_result = next(generator)\n  File \"E:\\Selenium_Demo-main\\TestCase\\test_001_baiduSearch.py\", line 25, in init\n    driver.maximize_window()\n  File \"C:\\Users\\bonjour\\AppData\\Local\\Programs\\Python\\Python39\\lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 737, in maximize_window\n    self.execute(command, params)\n  File \"C:\\Users\\bonjour\\AppData\\Local\\Programs\\Python\\Python39\\lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 321, in execute\n    self.error_handler.check_response(response)\n  File \"C:\\Users\\bonjour\\AppData\\Local\\Programs\\Python\\Python39\\lib\\site-packages\\selenium\\webdriver\\remote\\errorhandler.py\", line 242, in check_response\n    raise exception_class(message, screen, stacktrace)\n"}, "start": 1640938459806, "stop": 1640938460669}], "start": 1640938459806, "stop": 1640938461006}